packages/map-template/src/components/Map/GoogleMapsMap/GooglePlacesHandler.js

```diff
@@ -0,0 +1,23 @@
+let googleMapsGeocoder;
+
+/**
+ *  Handle google places.
+ *
+ * @param {Object} location
+ */
+function handleGooglePlaces(location) {
```

Comment:
This is a very generic name. Could we be more specific in the function name (and description) what it does?

Reply:
Could be something like `addGooglePlaceCoordinates`? Or do you have any suggestion? @ammapspeople

Reply:
What about `addGooglePlaceGeometry`?

Reply:
Renamed function ✅